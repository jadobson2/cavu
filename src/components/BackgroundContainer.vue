<template>
  <main class="px-8 xl:px-14 py-5 bg-gradient-to-r from-greyscale-80 to-black">
    <slot></slot>
  </main>
</template>

<script lang="ts">
import Vue from 'vue'

export default Vue.extend({
  name: 'BackgroundContainer'
})
</script>
