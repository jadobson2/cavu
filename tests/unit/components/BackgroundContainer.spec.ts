import { shallowMount } from '@vue/test-utils'
import BackgroundContainer from '@/components/BackgroundContainer.vue'

describe('BackgroundContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(BackgroundContainer, {
      slots: { default: '<div>Foo</div>' }
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
