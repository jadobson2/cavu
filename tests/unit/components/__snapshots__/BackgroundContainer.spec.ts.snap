// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundContainer renders correctly 1`] = `
<main
  class="px-8 xl:px-14 py-5 bg-gradient-to-r from-greyscale-80 to-black"
>
  <div>
    Foo
  </div>
</main>
`;
